modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/RunFromStep.scala

```diff
@@ -24,9 +24,10 @@ import web.client.style._
   * Contains the control to start a step from an arbitrary point
   */
 object RunFromStep {
-  final case class Props(id:      Observation.Id,
-                         stepId:  Int,
-                         runFrom: StartFromOperation)
+  final case class Props(id:               Observation.Id,
+                         stepId:           Int,
+                         resourceInFligth: Boolean,
```

Comment:
I think there is a typo here.  If so, what does "in flight" mean in this context?